src/Stan/Inspection/Partial.hs

```diff
@@ -0,0 +1,58 @@
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+Contains all 'Inspection's for partial functions.
+-}
+
+module Stan.Inspection.Partial
+    ( -- * Partial 'Inspection's
+      stan0001
+    , stan0001Inspection
+
+      -- * List of all partial 'Inspection's
+    , partialInspections
+    , partialInspectionsIds
+    ) where
+
+import Stan.Category (partial)
+import Stan.Core.Id (Id (..))
+import Stan.Inspection (Inspection (..), Severity (..))
+
+
+-- | All partial 'Inspection's.
+partialInspections :: [Inspection]
+partialInspections =
+    [ stan0001Inspection
+    ]
+
+-- | All partial 'Inspection's 'Id's.
+partialInspectionsIds :: [Id Inspection]
+partialInspectionsIds =
+    [ stan0001
+    ]
+
+-- | Smart constructor to create partial 'Inspection'.
+mkPartialInspection :: Id Inspection -> Text -> Text -> Inspection
```

Comment:
I imagine, we'll need a data type like:

```haskell
 -- TODO: better name
data NameLocation = NameLocation
    { package :: Text
    , moduleName :: Text
    , name :: Text
    }
```

And then such smart constructors will take a value of this data type as an argument. We also will need such data type for analysis. What do you think?